httplog.go

```diff
@@ -102,18 +102,18 @@ func Handler(logger *Logger, optSkipPaths ...[]string) func(next http.Handler) h
 }
 
 type requestLogger struct {
-	Logger  slog.Logger
+	Logger  *slog.Logger
 	Options Options
 }
 
 func (l *requestLogger) NewLogEntry(r *http.Request) middleware.LogEntry {
-	entry := &RequestLoggerEntry{}
+	entry := &RequestLoggerEntry{l.Logger, l.Options, ""}
```

Comment:
This is the most important fix, passing the l.Options here is important to honor options in all `RequestLoggerEntry` functions